patches/net/minecraft/client/resources/language/ClientLanguage.java.patch

```diff
@@ -1,14 +1,60 @@
 --- a/net/minecraft/client/resources/language/ClientLanguage.java
 +++ b/net/minecraft/client/resources/language/ClientLanguage.java
-@@ -34,6 +_,7 @@
+@@ -22,36 +_,50 @@
+ public class ClientLanguage extends Language {
+     private static final Logger LOGGER = LogUtils.getLogger();
+     private final Map<String, String> storage;
++    private final Map<String, net.minecraft.network.chat.Component> componentStorage;
+     private final boolean defaultRightToLeft;
+ 
++    @Deprecated
+     private ClientLanguage(Map<String, String> p_118914_, boolean p_118915_) {
++        this(p_118914_, Map.of(), p_118915_);
++    }
++
++    private ClientLanguage(Map<String, String> p_118914_, Map<String, net.minecraft.network.chat.Component> componentStorage, boolean p_118915_) {
+         this.storage = p_118914_;
++        this.componentStorage = componentStorage;
```

Comment:
This is a weird place to put the patch, maybe move it at the end?

Reply:
Huh? Should I move the added constructor to the end or the field set?

Reply:
The field set and the parameter in the added constructor.

Reply:
I think I did it?